Emby.Server.Implementations/ScheduledTasks/Tasks/AudioNormalizationTask.cs

```diff
@@ -76,81 +76,94 @@ public AudioNormalizationTask(
     /// <inheritdoc />
     public async Task ExecuteAsync(IProgress<double> progress, CancellationToken cancellationToken)
     {
-        foreach (var library in _libraryManager.RootFolder.Children)
+        var numComplete = 0;
+        var libraries = _libraryManager.RootFolder.Children.Where(library => _libraryManager.GetLibraryOptions(library).EnableLUFSScan).ToArray();
+        double percent = 0;
+
+        foreach (var library in libraries)
         {
-            var libraryOptions = _libraryManager.GetLibraryOptions(library);
-            if (!libraryOptions.EnableLUFSScan)
-            {
-                continue;
-            }
+            var albums = _libraryManager.GetItemList(new InternalItemsQuery { IncludeItemTypes = [BaseItemKind.MusicAlbum], Parent = library, Recursive = true });
 
-            // Album gain
-            var albums = _libraryManager.GetItemList(new InternalItemsQuery
-            {
-                IncludeItemTypes = [BaseItemKind.MusicAlbum],
-                Parent = library,
-                Recursive = true
-            });
+            double nextPercent = numComplete + 1;
+            nextPercent /= libraries.Length;
+            nextPercent -= percent;
+            nextPercent /= 2;
+            var albumComplete = 0;
 
             foreach (var a in albums)
             {
-                if (a.NormalizationGain.HasValue || a.LUFS.HasValue)
-                {
-                    continue;
-                }
-
                 // Skip albums that don't have multiple tracks, album gain is useless here
```

Comment:
This comment is now in a wrong place. It should now be put on the `if (albumTracks.Count > 1)` line.